CMakeLists.txt

```diff
@@ -320,25 +325,10 @@ if(UNIX AND NOT APPLE)
 endif()
 
 # -------------------------------------------------------------
-# check for some things with TCP core and boost
+# add some additional libraries for MSYS
 # -------------------------------------------------------------
 if(MSYS)
-    if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
-        if(BOOST_VERSION_LEVEL GREATER 65)
-            set(DISABLE_TCP_CORE FALSE)
-            target_link_libraries(helics_base INTERFACE wsock32 ws2_32)
-            set(DISABLE_IPC_CORE TRUE)
-        else()
-            set(DISABLE_TCP_CORE TRUE)
-            message(STATUS, "disabling TCP core |${DISABLE_TCP_CORE}|")
-            target_link_libraries(helics_base INTERFACE ws2_32)
-            set(DISABLE_IPC_CORE FALSE)
-        endif()
-
-    else()
-        target_link_libraries(helics_base INTERFACE wsock32 ws2_32)
-    endif()
-
+    target_link_libraries(helics_base INTERFACE wsock32 ws2_32)
```

Comment:
This was an issue with asio on MSYS, won't be an issue since we moved to standalone asio, though do need to double check that about the IPC core stuff